forge-game/src/main/java/forge/game/ability/AbilityKey.java

```diff
@@ -210,7 +211,7 @@ public static void addCardZoneTableParams(Map<AbilityKey, Object> map, CardZoneT
         map.put(AbilityKey.LastStateGraveyard, table.getLastStateGraveyard());
         map.put(AbilityKey.InternalTriggerTable, table);
     }
-    public static CardZoneTable addCardZoneTableParams(Map<AbilityKey, Object> map, forge.game.spellability.SpellAbility sa) {
+    public static CardZoneTable addCardZoneTableParams(Map<AbilityKey, Object> map, SpellAbility sa) {
```

Comment:
i don't know if that works

my IDE didn't wanted to import that, because the AbilityKey has a Key for SpellAbility already